client/src/controllers/KeyboardController.ts

```diff
@@ -54,6 +54,22 @@ export class KeyboardController extends Controller<
     }
   }
 
+  customKeyboardShortcutsEnabled(): boolean {
```

Comment:
Maybe we could consider using the Wagtail config approach instead?
See `client/src/config/wagtailConfig.ts` & `wagtail/admin/templates/wagtailadmin/admin_base.html` (`wagtail-config`) &  `wagtail/admin/templatetags/wagtailadmin_tags.py` (`wagtail_config` template tag)
.

And then just not register the controller at all or override it with an 'empty' controller, or at least import the config into the controller and use that way.